### ISSUE TITLE
A11y tables should be using compact1_written format instead of compact1

### ISSUE THREAD

**Issue by cookkate** [TBS-EACPD/infobase#232]
Probably? :)

Either way, it's a mix now, with `historical_g_and_c` using compact1_written and most (all?) other tables using compact1.
